### PR TITLE
Adding in data for patch34

### PR DESCRIPTION
**Pull request by WebSpider**

### CODE REVIEW

**Comment by WebSpider** [comment]
In order to modify the officers with traits, we need #84 first

File: officers/three_of_ten.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  },
+  "class": "science",
+  "faction": "neutral",
+  "group": "Unimatrix Twelve",
+  "img_url": "./assets/three_of_ten.png",
+  "officer_name": "Three of Ten",
+  "ranks": [
+    {
+      "name": "Ensign I",
+      "officer_ability_value": 1,
```
Comment (Iliasgx) [inline officers/three_of_ten.json:24]: Officer abilities should be 3.0, 3.5, 4.0, 4.5, 5.0

Comment (WebSpider) [reply officers/three_of_ten.json]: Fixed in 927a27

**Comment by WebSpider** [comment]
Currently missing: Away missions themselves. Will follow up in a seperate issue.

**Comment by Iliasgx** [comment]
The `level` field for each trait level should be an `int` instead of a `string` to make it conform other level indicators.

**Review by Iliasgx — APPROVED** [review]
We can add the critical chance info later. Lg

**Pull request merged**